Build of testthat fails

When I try to build the current master it fails with 
`
../inst/include/testthat/vendor/catch.h:6495:33: error: size of array ‘altStackMem’ is not an integral constant-expression

 6495 |         static char altStackMem[SIGSTKSZ];

size of array ‘altStackMem’ is not an integral constant-expression

 6546 |     char FatalConditionHandler::altStackMem[SIGSTKSZ] = {};

`
With gcc version: 11.1.1

--- a/NEWS.md
+++ b/NEWS.md
@@ -58,6 +58,9 @@
  * Support setting the testthat edition via an environment variable
   (`TESTTHAT_EDITION`) as well (@michaelquinn32, #1386).
  
+* The vendored Catch code used for `use_catch()` now uses an constant value for the stack size rather than relying on SIGSTKSZ.
+  This fixes compatibility for recent glibc versions where SIGSTKSZ is no longer a constant.
+
 # testthat 3.0.2
 
 * Failing expectations now include a backtrace when they're not called directly

--- a/NEWS.md
+++ b/NEWS.md
@@ -1,5 +1,8 @@
 # testthat (development version)
 
+* The vendored Catch code used for `use_catch()` now uses an constant value for the stack size rather than relying on SIGSTKSZ.
+  This fixes compatibility for recent glibc versions where SIGSTKSZ is no longer a constant.
+
 * Condition expectations now consistently return the expected
   condition instead of the return value (#1371). Previously, they
   would only return the condition if the return value was `NULL`,
@@ -58,9 +61,6 @@
  * Support setting the testthat edition via an environment variable
   (`TESTTHAT_EDITION`) as well (@michaelquinn32, #1386).
  
-* The vendored Catch code used for `use_catch()` now uses an constant value for the stack size rather than relying on SIGSTKSZ.
-  This fixes compatibility for recent glibc versions where SIGSTKSZ is no longer a constant.
-
 # testthat 3.0.2
 
 * Failing expectations now include a backtrace when they're not called directly

